access-token-management/src/AccessTokenManagement/DPoP/DPoPProofString.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Duende.AccessTokenManagement.DPoP;
 
-public readonly record struct DPoPProofString : IStronglyTypedString<DPoPProofString>
+public readonly record struct DPoPProofString : IStronglyTypedValue<DPoPProofString>
```

Comment:
I think getting rid of the String suffix in other places is good. We still have this DPoPProofString though (since there was also DPoPProof). I think we could possibly get rid of that last String name too - we just need a different name for DPoPProof. Looking at the usages, maybe we should go back to DPoPProofRequest?

So...
DPoPProof -> DPoPProofRequest and
DPoPProofString -> DPoPProof

I think I originally recommended renaming DPoPProofRequest, so sorry about that - I had to see it to this way realize what I wanted I guess!

Reply:
As discussed, I refactored the code to no longer need this request.